_sass/color_schemes/memo.scss

```diff
@@ -23,19 +23,19 @@ $grey-dk-200: #797e86; // grayscale tertiary
 $blue-300: #0d4189; // blue dark
 $blue-200: #0b5dd0; // blue memo
 $blue-000: #f0f6fe; // blue light
-$sidebar-color: $blue-000;
+$sidebar-color: $grey-dk-300;
 $body-heading-color: $grey-dk-300;
 $body-text-color: $grey-dk-250;
 $link-color: $blue-200;
 $btn-primary-color: $blue-200;
-$feedback-color: darken($sidebar-color, 3%);
+$feedback-color: darken($sidebar-color, 0%);
```

Comment:
This makes the background color of search results very dark. I'll add a custom rule on top of that to circumvent this issue.